### PR TITLE
Fix: implement endpoint for retrieving chart data

### LINKED ISSUE #140: Add endpoint for user metrics

### PR DESCRIPTION
**Pull request by jkarenzi**
### What does this PR do?
- This PR implements endpoints for retrieving chart data

### Description of the task to be completed?
- [x] Implement endpoint for fetching buyer and vendor chart data
- [x] Implement endpoint for fetching category chart data

### How can this be manually tested?
- By accessing the api/v1/user/get_metrics endpoint for buyer and vendor chart data
- By accessing the api/v1/category/get_metrics endpoint for category chart data

### What are the relevant pivotal trackers/story id?
- #140

### CODE REVIEW

File: src/middlewares/errorHandler.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ function errorHandler(func: MiddlewareFunction): MiddlewareFunction {
     try {
       return await func(req, res);
     } catch (error) {
+      console.log(error)
```
Comment (13XAVI) [inline src/middlewares/errorHandler.ts:13]: Rmove console

File: src/controller/userController.ts [hunk 2]
```diff
@@ -413,3 +413,21 @@ export const removeProfileImg = errorHandler(
...
+  for(const user of users){
+    const monthIndex = user.createdAt.getMonth()
+    if(user.userType.name === 'Buyer'){
+      buyerData[monthIndex] += 1
+    }else if(user.userType.name === 'Vendor'){
+      vendorData[monthIndex] += 1
+    }
+  }
+
+  return res.status(200).json({buyerData, vendorData})
```
Comment (13XAVI) [inline src/controller/userController.ts:432]: remove console

**Review by 13XAVI — COMMENTED** [review]
Remove all consoles

**Review by ambroisegithub — APPROVED** [review]
It looks good for merge

**Review by 13XAVI — APPROVED** [review]
Nice

**Review by 13XAVI — APPROVED** [review]

**Review by 13XAVI — APPROVED** [review]

**Review by EddyShimwa — APPROVED** [review]
👍

**Review by wayneleon1 — APPROVED** [review]
Well done

**Review by dushimeemma — APPROVED** [review]

**Pull request merged**